bower.json

```diff
@@ -22,6 +22,11 @@
     "test",
     "tests"
   ],
+  "dependencies": {
+    "angular": "~1.4.4",
+    "angular-sanitize": "~1.4.4",
```

Comment:
By the way if your app is not an Angular app you can use pure JS version of this mofule. Here you can find it here:
https://github.com/mohsen1/json-formatter-js